wandb/sdk/interface/interface.py

```diff
@@ -33,6 +32,7 @@
 from . import summary_record as sr
 from .artifacts import ArtifactManifest
 from .message_future import MessageFuture
+from ..data_types.converters import history_dict_to_json, val_to_json
```

Comment:
maybe better name would be `data_types.utils` or something like this will allow to add other utility functions that are specific for data_types?